mesonbuild/cargo/interpreter.py

```diff
@@ -153,10 +152,8 @@ def _add_dependency(self, pkg: PackageState, depname: str) -> None:
             return
         dep = pkg.manifest.dependencies.get(depname)
         if not dep:
-            if depname in itertools.chain(pkg.manifest.dev_dependencies, pkg.manifest.build_dependencies):
-                # FIXME: Not supported yet
-                return
-            raise MesonException(f'Dependency {depname} not defined in {pkg.manifest.package.name} manifest')
+            # It could be build/dev/target dependency. Just ignore it.
+            return
```

Comment:
I take it it's impossible to get the target deps here?

Reply:
You get here for a disabled target dep. I will clarify the comment.